src/emulator/controller.ts

```diff
@@ -305,7 +305,11 @@ function findExportMetadata(importPath: string): ExportMetadata | undefined {
   }
 }
 
-export async function startAll(options: Options, showUI: boolean = true): Promise<void> {
+interface EmulatorOptions extends Options {
+  extensionEnv?: Record<string, string>;
```

Comment:
This probably needs to be changed. Previously, when emulating a single, local extension via the `ext:emulators:*` commands, we would pass in the param values  and the extensionTriggers via Options. At the time, this made (some) sense - it kept a nice separation between the extensions specific code and the general functions emulator, and options was an Any, so it didn't seem like a big deal to throw in an extra field or two. 

Now, we've actually typed Options and decided that it represents things passed in via flags, and suddenly extensionEnv is a poor fit. I created this EmulatorOptions type to avoid polluting the global Options type further - however, it might be better to move these to their own type and change the signature of startAll - something like:
```
interface LocalExtensionInfo {
  triggers: ParsedTriggerDefinition[],
  env: Record<string, string>
}

export async function startAll(options:Options, showUI: boolean = true, extensionInfo?: LocalExtensionInfo ) {...}
```

Further down the line, I'd like to change how the `ext:emulators:*` commands work entirely and handle local extensions in the same way we do all other extensions.